### PR TITLE
[SST-17325] Conditions index

### PR DESCRIPTION
**Pull request by pluus**
### Description
Introducing `/api/v3/conditions` endpoint.

https://tractionguest.atlassian.net/browse/SST-17325

### Changelog
- Added Experience.v1.yaml
- Added Condition.v1.yaml
- Added ConditionList.v1.yaml
- Added Conditions.v1.yaml
- Added /conditions endpoint
- Added limit/offset query params to the endpoint
- Added 200/400/401 response messages.

## Notes (can delete this section after doing):
- [ ] Start at least one of your commits with your wrike ticket in the following format, `[#1234] Your concise and explicit commit message`
- [ ] Use the `Squash and Merge` button to combine your commits in to one final commit (with the wrike ticket) to go in to master

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -64,6 +64,13 @@ paths:
                 $ref: ./models/ErrorsList.v1.yaml
       operationId: getBatch
       description: Retrieve a given `BatchJob` entity.
+      parameters:
+        - schema:
+            type: string
+          in: query
+        - schema:
+            type: string
+          in: query
```
Comment (cpb) [inline openapi.yml:73]: Was this intentional?

File: openapi.yml [hunk 2]
```diff
@@ -211,6 +218,12 @@ paths:
           schema:
             type: string
           in: query
+        - schema:
+            type: string
+          in: query
+        - schema:
+            type: string
+          in: query
```
Comment (cpb) [inline openapi.yml:226]: Was this intentional?

File: openapi.yml [hunk 3]
```diff
@@ -4213,6 +4226,43 @@ paths:
...
+      tags:
+        - Conditions
+      responses:
+        '200':
+          description: Successful response - returns an array of `Conditions`.
+        '400':
+          description: A generic error
+        '401':
+          description: You don't have permission to view this
+      operationId: get-api-v3-conditions
```
Comment (cpb) [inline openapi.yml:4241]: I've seen us use `getApiV3Conditions` case else where… I think …

File: openapi.yml [hunk 4]
```diff
@@ -4213,6 +4226,43 @@ paths:
...
+      parameters:
+        - schema:
+            type: string
+          in: query
+          name: limit
+          description: Limits the results to a specified number. Defaults to 50.
+        - schema:
+            type: string
+          in: query
+          name: offset
```
Comment (cpb) [inline openapi.yml:4263]: Yyyyyeaaassss -- the `after_id` one won't work here

**Review by cpb — APPROVED** [review]
Only slight confusion about the naming of Condition, Conditions, ConditionList but I think we can get by

**Comment by gregunderwood** [comment]
This is no longer relevant as this endpoint was deprecated in favour of getting conditions from a v3 experiences endpoint

**Pull request closed without merging**